cal/cal/static/js/app.js

```diff
@@ -111,29 +111,41 @@ analyticsApp.component('tags', {
 function TagsDetailCtrl($scope, $http) {
   var tagUrl = '/v1/tags/' + this.tagId + '/events';
   var eventweek = '/v1/tags/' + this.tagId + '/eventWeek';
+  var query_timezone = moment.tz.guess();
   $scope.tagDetails = [];
   $scope.tagEvents = [];
+  $scope.tagHours = this.tagHours;
```

Comment:
Notice that in the configuration of your component you have `controllerAs: '$ctrl'`, which means that you should refer to your variable using `$ctrl.tagDetails`. I believe that should solve the issue